devtools/clean_workspace.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -o errexit -o nounset -o pipefail
+command -v shellcheck >/dev/null && shellcheck "$0"
+
+for contract_dir in contracts/*/
+do
+  (cd "$contract_dir" && cargo clean)
+done
+
+(cd packages/std && cargo clean)
+(cd packages/crypto &&  cargo clean)
+(cd packages/storage &&  cargo clean)
+(cd packages/schema &&  cargo clean)
+(cd packages/vm &&  cargo clean)
```

Comment:
Could you run `./devtools/format_sh.sh` to format those?

Reply:
I _could_, but I don't want to install nodejs and a zillion packages, just to format a shell script. Isn't there a cheap way to have this utility up and running?

Reply:
For shell formatting, shfmt is used. This does not require anything from the nodejs ecosystem. I get it via homebrew. It is available in some linux package managers, like here https://archlinux.org/packages/community/x86_64/shfmt/. Looks like it is distributed via snap on Ubuntu: https://snapcraft.io/install/shfmt/ubuntu

Reply:
Thanks! When I searched for it months ago, I thought it was a node app.

Reply:
Yeah, there is this prettier plugin that integrates shell formatting into the node.js ecosystem. A lot of wrapping we don't need. See https://github.com/CosmWasm/cosmwasm/issues/652#issuecomment-740510520

Reply:
It's a go util, and (months ago) I didn't have go installed. Later I thought it was nodejs stuff.

Installing it is as simple as doing `go get -u github.com/mvdan/sh/cmd/shfmt` (and putting $HOME/go/bin in your PATH).